test/api/attributes.js

```diff
@@ -359,7 +370,7 @@ describe('$(...)', function () {
         balls: 'giandor',
       });
       // Adding as string.
-      var b = $('.linth').data('snack', 'chocoletti');
+      var b = $('.linth').data('snack', 'chocoletti').data(4, 'chocoletti');
```

Comment:
Check the value of `data(4)`?

Reply:
yes I used not a string for test. and it doesnt break.

It managed to throw when I did same with setAttr.

Reply:
I added throw check